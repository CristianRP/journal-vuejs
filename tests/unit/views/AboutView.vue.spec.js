import { shallowMount } from '@vue/test-utils'
import AboutView from '@/views/AboutView'

describe('Tests on About View component', () => {
  test('should match with the snapshot', () => {
    const wrapper = shallowMount(AboutView)
    expect( wrapper.html ).toMatchSnapshot()
  })
});
