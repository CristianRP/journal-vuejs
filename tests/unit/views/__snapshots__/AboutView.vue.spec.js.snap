// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests on About View component should match with the snapshot 1`] = `[Function]`;
